cvmfs/hash.h

```diff
@@ -62,14 +65,15 @@ const unsigned kMaxDigestSize = 32;
  * for backwards compatibility MD5 and SHA-1 have none.
  */
 extern const char *kAlgorithmIds[];
-// in hash.cc: const char *kAlgorithmIds[] = {"", "", "-rmd160", "-sha256", ""};
-const unsigned kAlgorithmIdSizes[] = {0, 0, 7, 7, 0};
-const unsigned kMaxAlgorithmIdentifierSize = 7;
+// in hash.cc: const char *kAlgorithmIds[] = {"", "", "-rmd160", "-sha256", ...
+const unsigned kAlgorithmIdSizes[] = {0, 0, 7, 7, 5, 9, 0};
+const unsigned kMaxAlgorithmIdentifierSize = 9;
```

Comment:
C++11/14: not exactly sure about that, but could that be determined with constexpr in C++14 at compile time?